src/main/groovy/nebula/plugin/info/dependencies/DependenciesInfoPlugin.groovy

```diff
@@ -39,12 +44,17 @@ class DependenciesInfoPlugin implements Plugin<Project>, InfoCollectorPlugin {
                                         return m1.name.compareTo(m2.name) // name is required
                                     versionComparator.compare(m1.version, m2.version)
                                 })*.toString().join(',')
-
                     if (resolvedDependencies) {            
                         manifestPlugin.add("Resolved-Dependencies-${it.name.capitalize()}", resolvedDependencies)
+                        dependencies.put("Resolved-Dependencies-${it.name.capitalize()}", resolvedDependencies)
                     }
                 }
             }
+
+            dependencyMap["${project.name}-dependencies".toString()] = dependencies
+            if (project == project.rootProject) {
```

Comment:
there will be one root project, need to collect them in all of the subprojects only publish in root